SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -526,23 +522,23 @@ private void Benevolence()
                 .Name("Benevolence")
 
                 .AddPerkLevel()
-                .Description("Restores 40 HP to a single target.")
+                .Description("Restores 30 + Willpower Mod * 4 + (Willpower Mod*4)d10 health to target. If targeting another, healing is doubled, cost is increased by 10 FP. Willpower Mod/2 stamina is taken (bonuses not lost on insufficient stamina), to restore Willpower Mod/2 Stamina and FP to target and provide Willpower Mod * 2 regen for 4 rounds.")
                 .Price(2)
                 .RequirementSkill(SkillType.Force, 10)
                 .RequirementCharacterType(CharacterType.ForceSensitive)
                 .RequirementCannotHavePerk(PerkType.CreepingTerror)
                 .GrantsFeat(FeatType.Benevolence1)
 
                 .AddPerkLevel()
-                .Description("Restores 80 HP to a single target.")
+                .Description("Restores 60 + Willpower Mod * 4 + (Willpower Mod*4)d10 health to target. If targeting another, healing is doubled, cost is increased by 10 FP. Willpower Mod/2 stamina is taken (bonuses not lost on insufficient stamina), to restore Willpower Mod/2 Stamina and FP to target and provide Willpower Mod * 2 regen for 4 rounds.")
```

Comment:
This is a good example of what I meant earlier about simplicity. My eyes glazed over just reading this description. ;)